napari/components/layerlist.py

```diff
@@ -184,6 +184,21 @@ def _extent_world(self) -> np.ndarray:
         """
         return self._get_extent_world([layer.extent for layer in self])
 
+    @cached_property
```

Comment:
We need to clear the cached value for this when `Points.size` changes, but I don't see that yet. Apologies if I missed it! Could also be a good opportunity for test coverage since caches can always cause problems.

Reply:
I think the most general way to do this would be to check whether `extent_agumented` changed. What do you think: should layers have a new (maybe private) event called `extent_augmented` or rather simply fire the `extent` when for example when `Points.size` changes?

Reply:
The `extent_augmented` event sounds good to me, since that can change without `extent` changing (and `Points.size` changing won't actually change `extent` either). Then the layer list can also subscribe to that as it does to the `extent` event.

Since `Layer.extent_augmented` is public, I'd probably make the event public too. Or I'd make both private.